main_test.go

```diff
@@ -25,3 +26,35 @@ func (t *MainTest) TestCreateStorageHandleEnableStorageClientLibraryIsTrue() {
 	ExpectNe(nil, storageHandle)
 	ExpectEq(nil, err)
 }
+
+func (t *MainTest) TestCreateStorageHandle() {
+	flags := &flagStorage{
+		DisableHTTP2:        false,
+		MaxConnsPerHost:     5,
+		MaxIdleConnsPerHost: 100,
+		HttpClientTimeout:   5,
+		MaxRetryDuration:    7,
+		RetryMultiplier:     2,
+		AppName:             "app",
+	}
+
+	storageHandle, err := createStorageHandle(flags)
+
+	AssertEq(nil, err)
+	AssertNe(nil, storageHandle)
+}
+
+func (t *MainTest) TestGetUserAgentWhenEnvironmentVariableIsSet() {
```

Comment:
call out the envvariable name here. If we are dealing with multiple env variable names, it will become confusing.

Reply:
fixed